### PR TITLE
ZEN-31009 Moving devices to/from Groups, Locations and Systems result…

### PR DESCRIPTION
**Pull request by romanbakaleyko**
… in 'Remodel Required' modal (#3478)

(cherry picked from commit 2be4042b7da3c2beb74ef4158890eb3bbff33e58)

### CODE REVIEW

**Comment by romanbakaleyko** [comment]
forward port to develop https://github.com/zenoss/zenoss-prodbin/pull/3478

File: Products/Zuul/facades/devicefacade.py [hunk 1]
```diff
@@ -558,30 +558,38 @@ def _moveDevices(self, uids, target):
         devs = (self._getObject(uid) for uid in uids)
         targetname = target.getOrganizerName()
         moved_devices_count = 0
+        success = False
```
Comment (jpeacock-zenoss) [inline Products/Zuul/facades/devicefacade.py]: Is the `success` variable necessary?  I don't see any changes that read the result of `success`.  The Javascript code appears to only care about the `remodel_required` flag.

Comment (romanbakaleyko) [reply Products/Zuul/facades/devicefacade.py]: https://github.com/zenoss/zenoss-prodbin/pull/3536/files#diff-b492fee73489e9315626c38afd358b04R994

Comment (jpeacock-zenoss) [reply Products/Zuul/facades/devicefacade.py]: I see now.  I wasn't looking at existing code before.  Only the changed lines.

**Pull request merged**